mdtraj/formats/pdb/pdbfile.py

```diff
@@ -211,6 +215,10 @@ class PDBTrajectoryFile(object):
     force_overwrite : bool
         If opened in write mode, and a file by the name of `filename` already
         exists on disk, should we overwrite it?
+    standard_names : bool, default=True
+        If set to false, non-standard atomnames and residuenames will NOT be
```

Comment:
Can you change the order of the clauses in this sentence? Start with 'If True, we standardize..., If False, this step is skipped.'